.gitignore

```diff
@@ -1,5 +1,6 @@
+.DS_Store
 node_modules
 /.tern-port
 /local
-/bin/acorn
+/bin/acorn.js
```

Comment:
Shouldn't that be `/bin/_acorn.js`?

Reply:
I'm going to assume it should, and updated patch 2b6ed51 in that way